plugins/processors/dcos_metadata/dcos_metadata.go

```diff
@@ -97,7 +97,7 @@ func (dm *DCOSMetadata) Apply(in ...telegraf.Metric) []telegraf.Metric {
 
 	if stale {
 		cids := []string{}
-		for cid, _ := range unknown {
+		for cid := range unknown {
```

Comment:
Not relevant for this commit but it looks weird for me to range for the unknown in order to get some data.